src/menus/FileMenus.cpp

```diff
@@ -184,6 +186,41 @@ void OnOpen(const CommandContext &context )
 {
    auto &project = context.project;
    ProjectManager::OpenFiles(&project);
+
+   int unavailablePlugins = 0;
+
+   auto &trackList = TrackList::Get(project);
+   for (auto track : trackList.Leaders())
+   {
+      //Avoid creation-on-demand of a useless, empty list in case the track is of non-wave type.
+      if (dynamic_cast<WaveTrack*>(track) == nullptr)
+         continue;
+
+      auto& effects = RealtimeEffectList::Get(*track);
+      effects.Visit([&unavailablePlugins](auto& state, bool)
+         {
+            const auto& ID = state.GetID();
+            const PluginDescriptor* plug = PluginManager::Get().GetPlugin(ID);
+
+            if (plug)
+            {
+               if (!PluginManager::IsPluginAvailable(*plug))
+                  unavailablePlugins++;
+            }
+            else
+            {
+               unavailablePlugins++;
+            }
+         });
+   }
+
+   if (unavailablePlugins > 0)
+   {
+      BasicUI::ShowMessageBox(
+         XO("This project contains some plugins that cannot be found on the system."),
+         BasicUI::MessageBoxOptions()
+            .Caption(XO("Project Error")));
```

Comment:
Let's do
> Window title: Missing Plugins
> This project contains some realtime effect plugins that cannot be found on this system.
> The project may sound different than intended. [Learn more](https://audacityteam.org/errors#missing-plugins)
> [OK]

with the "learn more" having an accessible link to https://audacityteam.org/errors#missing-plugins